.github/workflows/build-natives.yaml

```diff
@@ -75,6 +75,29 @@ jobs:
           for /f "tokens=3 delims=<> skip=1" %%v in ('findstr "<version>" ..\main\pom.xml') do set WILDFLY_OPENSSL_NATIVES_VERSION=%%v
           echo Running tests with WildFly OpenSSL Natives %WILDFLY_OPENSSL_NATIVES_VERSION%
           mvn -B verify --file pom.xml -Dorg.wildfly.openssl.path="C:\vcpkg\installed\x64-windows\tools\openssl" -Dversion.org.wildfly.openssl.natives=%WILDFLY_OPENSSL_NATIVES_VERSION%
+      - if: matrix.os == 'ubuntu-latest'
+        name: Install latest version of OpenSSL in ubuntu
+        run: |
+          cd /usr/local/src/
+          sudo wget https://www.openssl.org/source/openssl-3.0.7.tar.gz
+          sudo tar -xf openssl-3.0.7.tar.gz
+          cd openssl-3.0.7
+          sudo ./config --prefix=/usr/local/ssl --openssldir=/usr/local/ssl shared zlib
+          sudo make
+          sudo make test
+          sudo make install
+          cd /etc/ld.so.conf.d/
+          echo "/usr/local/ssl/lib64" | sudo tee openssl-3.0.7.conf
+          sudo ldconfig -v
+      - if: matrix.os == 'ubuntu-latest'
+        name: Run tests with the built Linux native
```

Comment:
Are we missing a block for building the Mac native with the newly installed OpenSSL?

Reply:
Yes, sorry I realized it a few minutes ago , it was built with the old version and then wilfdly-openssl tested with new one I think, working on it!

Reply:
macos is almost ready as well, will update both

Reply:
Awesome!